php-checks/src/test/resources/checks/DuplicatedMethodCheck.php

```diff
@@ -97,3 +97,28 @@ public function emptyMethod() {
     public function isEmptyMethod() {
     }
 }
+
+function myFunction() : A
+{
+    return null;
+}
+
+function myFunction() : array
+{
+    return null;
+}
+
+function myFunction($a, $lot, $of, $params) : array
+{
+    return null;
+}
+
+function myFunction(int $a, bool $lot, $of, $params) : array // Noncompliant
```

Comment:
What happens if you add another function? `myFunction(string $a, bool $lot, $of, $params) : array`?
Would it also get flagged? (According to the logic I see it should)

Reply:
Yes, as we are now checking only the number of parameters. This was a part of the ticket description and I think it makes sense considering how not strictly PHP allows to set parameter types